src/renderer/features/operations/OperationsValidation/model/transfer-validate-model.ts

```diff
@@ -0,0 +1,147 @@
+import { Store, createEffect, createEvent, sample } from 'effector';
+import { ApiPromise } from '@polkadot/api';
+
+import { Asset, Balance, Chain, ID, Transaction } from '@shared/core';
+import {
+  AccountStore,
+  AmountFeeStore,
+  SignatoryFeeStore,
+  TransferRules,
+} from '@features/operations/OperationsValidation/lib/transfer-rules';
+import { getAssetById, toAccountId, transferableAmount } from '@shared/lib/utils';
+import { balanceModel, balanceUtils } from '@entities/balance';
+import { BalanceMap, NetworkStore } from '@widgets/Transfer/lib/types';
+import { ValidationResult, applyValidationRules } from '@features/operations/OperationsValidation';
+import { networkModel } from '@entities/network';
+import { transactionService } from '@entities/transaction';
+
+const validationStarted = createEvent<{ id: ID; transaction: Transaction }>();
+const txValidated = createEvent<{ id: ID; result: ValidationResult }>();
+
+type ValidateParams = {
+  id: ID;
+  api: ApiPromise;
+  chain: Chain;
+  asset: Asset;
+  transaction: Transaction;
+  balances: Balance[];
+};
+
+const validateFx = createEffect(async ({ id, api, chain, asset, transaction, balances }: ValidateParams) => {
+  const accountId = toAccountId(transaction.address);
+
+  const fee = await transactionService.getTransactionFee(transaction, api);
+
+  const rules = [
+    {
+      value: transaction.address,
+      form: {},
+      ...TransferRules.account.noProxyFee({} as Store<AccountStore>),
+      source: {
+        fee,
+        // TODO: Add support proxy
```

Comment:
When we need to support proxies in basket this needs to be somehow addressed, right?